src/transaction-runner.coffee

```diff
@@ -217,4 +220,16 @@ class TransactionRunner
         configuration.emitter.emit 'test error', error, test if error
         return callback()
 
+  isMultipart: (requestOptions) =>
+    caseInsensitiveRequestHeaders = {}
+    for key, value of requestOptions.headers
+      caseInsensitiveRequestHeaders[key.toLowerCase()] = value
+    caseInsensitiveRequestHeaders['content-type']?.indexOf("multipart") > -1
```

Comment:
I'd love to use https://www.npmjs.com/package/caseless here.

`caseless(requestOptions.headers).get('content-type')?.indexOf('multipart') > -1` ... but I can see we might refactor more parts of it soon.